src/js/base/fetch.js

```diff
@@ -1,9 +1,79 @@
 //  Similar to https://github.com/akre54/Backbone.Fetch
-import { isObject, isArray, defaults, extend, map, flatten, reduce, first, rest } from 'underscore';
+import { isObject, isArray, defaults, extend, map, flatten, reduce, first, rest, get } from 'underscore';
 import Radio from 'backbone.radio';
 
 import { logResponse } from 'js/datadog';
 
+const fetchers = [];
+
+function registerFetcher(baseUrl, fetcher, controller) {
```

Comment:
The "control" file for registering fetchers was moved into the base fetcher to simplify how the abort controller is understood since it can't reliably be attached to the fetcher itself.